dashboard/app/models/pd/workshop_material_order.rb

```diff
@@ -47,7 +53,10 @@ class WorkshopMaterialOrder < ActiveRecord::Base
     validates_presence_of :zip_code
     validates_presence_of :phone_number
     validates_inclusion_of :state, in: STATE_ABBR_WITH_DC_HASH.keys.map(&:to_s), if: -> {state.present?}
-    validates :phone_number, us_phone_number: true, if: -> {phone_number.present?}
+
+    # Make sure the phone number contains at least 10 digits.
+    # Allow any format and additional text, such as extensions.
+    validates :phone_number, format: /(\d.?){10}/, if: -> {phone_number.present?}
```

Comment:
Also this just makes sure the 10 digits appears somewhere in the string, but does not match the whole string. So something like `blah blah 1234567890` would also match and pass the validation in our case, starting at the first digit.